### DISCUSSION TITLE
Welcome to Obsidian-Obsidianite Discussions!

### DISCUSSION THREAD

**Discussion by bennyxguo** [bennyxguo/Obsidian-Obsidianite#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kkphoenixgx** [comment]
Hey, my name is kkphoenix, an user of our obsidian theme from Brazil . I would love to see different colors for each h tag that I have in my  notes. Because I just love our theme, there is no one that is equivalent, but is hard to me to understand where I am with a lot of headers in my notes in the same color. Just a simple ideia that will change my life ^^.
